Added blog post about SIG OBS TL. (#24)
bwplotka/my: web/content/posts/sig-obs-tl.md

project! You can use SIG Observability space for this freely, so [reach us!](#how-you-can-reach-us)

### Be Open Minded for Outside World

One point I want to highlight is that, as the SIG Observability, we are not limited to a few hosted projects only. There are certainly some gaps
in cloud native observability portfolio. Things that we could learn and project that already comply, integrate or even are already extremely useful for
complete observability journey in the CNCF ecosystem but not directly under CNCF governance. I am looking at you: [Grafana](https://github.com/grafana/grafana),
[Loki](https://github.com/grafana/loki) and [ConProf](https://github.com/conprof/conprof)!

It would be awesome to maintain a good relationship with such projects, allow even better integration with them, communicate more, and help each other!
 
> **Action Item**: If you feel that some projects or initiatives are extremely useful, but missing in Observability the CNCF portfolio, or something we can learn [reach us as well!](#how-you-can-reach-us)

## Summary

![View from Pyrgos village on Santorini island, Greece](/images/blog/sig-obs-tl/santorini2.jpg)

> I hope with this blog post, you learned a bit what this magic `SIG` concept is all about and how `YOU` can help within the area you are

Reviewer: ```suggestion
I hope with this blog post, you have learned a bit what this magical `SIG` concept is all about. And how `YOU` can help within the domain you are
```

> passionated the most about as well!

Reviewer: ```suggestion
passionated about!
```

> Thanks [all who voted for me](https://lists.cncf.io/g/cncf-toc/topic/76717430#5272) and particularly thanks to [Richi](https://twitter.com/twitchih)

Reviewer: ```suggestion
Thanks to [all who voted for me](https://lists.cncf.io/g/cncf-toc/topic/76717430#5272) and especially thanks to [Richi](https://twitter.com/twitchih)
```
